### PR TITLE
Minor classic refresh for 2024

### PR DESCRIPTION
**Pull request by roscopeco**
* Add DC jack
* Silk cleanup
* Unify added memory expansion headers
* Minor routing cleanup
* QR code on silk

### CODE REVIEW

File: design/r2020/kicad/rosco_m68k.kicad_pcb
Comment (0xTJ) [inline design/r2020/kicad/rosco_m68k.kicad_pcb:1]: The barrel jack center-positive symbol seems to be missing some parts on just this board. On the others, I see the whole thing, but on the r2020 I only see the + and -.

**Review by 0xTJ — CHANGES_REQUESTED** [review]
Overall, looks good! The only issue I see is on the silkscreen of r2020.

File: design/r2020/kicad/rosco_m68k.kicad_pcb
Comment (roscopeco) [reply design/r2020/kicad/rosco_m68k.kicad_pcb:1]: Huh, weird, thanks for spotting that! 

Should be there now :)

**Review by 0xTJ — APPROVED** [review]
Looks good!

**Comment by roscopeco** [comment]
> Looks good!

Thanks @0xTJ !

**Pull request merged**